package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "lint": "jshint .",
     "test": "mocha",
-    "install": "node proxy/build-client.js",
-    "rebuild-client": "node proxy/build-client.js"
+    "rebuild-build": "node scripts/build.js",
```

Comment:
Probably don't need to change the script name, just the file it is calling. If anyone has any sort of dev scripts set up, renaming this could break them.